ramls/instanceprecedingsucceedingtitle.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Is it possible to name file with `-` separator, e.g. `instance-preceding-succeeding-title.json` or similar?

Reply:
Yes, it is. In my opinion it is more readable also. However, all other json files use this style. And I am not sure it is good idea to use another style only for preceding-succeeding-titles json file.

Reply:
Well, yes. Let's keep naming in the same way for now. 

(I think we should have a tech debt to fix all the names).